R_scripts/licordat.R

```diff
@@ -14,15 +14,27 @@ read_licor_data <- function(filename) {
   r2 <- file[grepl("^Lin_R2:", file)]
   nobs <- length(file[grepl("^Obs#:", file)])
   date <- file[which(grepl("^Type", file)) + 1]
-  temp <- file[grepl("^Comments:", file)]
+  temp20 <- file[grepl("^Comments:", file)]
+
+  tablestarts <- grep("^Type", file)
+  tablestops <- grep("^CrvFitStatus", file)
+  tcham <- matrix()
+  t5 <- matrix()
+  
+  for (i in 1:length(tablestarts)) {
```

Comment:
`for (i in seq_along(tablestarts)) {` is better to use, in case `length(tablestarts)` is zero.